rust/agents/relayer/src/msg/pending_message.rs

```diff
@@ -322,7 +353,14 @@ impl PendingOperation for PendingMessage {
                 message_id=?self.message.id(),
                 "Transaction attempting to process message either reverted or was reorged"
             );
-            self.on_reprepare()
+            let span = info_span!(
```

Comment:
curious why this span but not for any other cases? wonder if these would already be in a span / if this should be for the other cases too

but probably a good idea not to instrument return values bc i'm sure
```
        if !self.is_ready() {
            return PendingOperationResult::NotReady;
        }
```
gets called a bunch and would be noisy

Reply:
ah so it's because I wanted to remove the `warn` on the line above, but the `warn` had two fields in the logs which I can't easily pass to `on_reprepare`. So I figured I'd just include those two fields in the span, and the span requires there to also be a string description, and if a warning ends up being printed in `on_reprepare`, those two fields will also be visible. I pushed a change that better reflects this now

Reply:
and to clarify, indeed the span doesn't show up when values are returned, just when logs are emitted

Reply:
ah cool, sgtm